test/src/integration/integration_test.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 var terraformDir = flag.String("directory", "", "Directory path of the terraform module to test")
+var cloudProvider = flag.String("cloud", "aws", "Cloud provider")
```

Comment:
```suggestion
var cloudProvider = flag.String("cloud_provider", "aws", "Cloud provider")
```
for consistency with the variable and ansible.

Reply:
Fixed in 7e1e078.